qa/rpc-tests/wallet_import_export.py

```diff
@@ -3,26 +3,45 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://www.opensource.org/licenses/mit-license.php .
 
+from test_framework.authproxy import JSONRPCException
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal, assert_true, start_nodes
 
 class WalletImportExportTest (BitcoinTestFramework):
     def setup_network(self, split=False):
         num_nodes = 3
-        extra_args = [["-exportdir={}/export{}".format(self.options.tmpdir, i)] for i in range(num_nodes)]
+        extra_args = [([
+            "-exportdir={}/export{}".format(self.options.tmpdir, i), 
+            ] + (["-walletrequirebackup"] if i == 0 else [])) for i in range(num_nodes)] 
         self.nodes = start_nodes(num_nodes, self.options.tmpdir, extra_args)
 
     def run_test(self):
         sapling_address2 = self.nodes[2].z_getnewaddress('sapling')
         privkey2 = self.nodes[2].z_exportkey(sapling_address2)
         self.nodes[0].z_importkey(privkey2)
 
+        # test walletconfirmbackup
+        try:
+            self.nodes[0].getnewaddress()
+        except JSONRPCException as e:
+            errorString = e.error['message']
+        assert_equal("Error: Please acknowledge that you have backed up" in errorString, True)
+        try:
+            self.nodes[0].z_getnewaddress('sapling')
+        except JSONRPCException as e:
+            errorString = e.error['message']
+        assert_equal("Error: Please acknowledge that you have backed up" in errorString, True)
+        dump_path0 = self.nodes[0].z_exportwallet('walletdumpmnem')
+        (mnemonic, t_keys0, sprout_keys0, sapling_keys0) = parse_wallet_file(dump_path0)
```

Comment:
Ignore the fields we aren't using yet.
```suggestion
        (mnemonic, _, _, _) = parse_wallet_file(dump_path0)
```